museum/sampleRecords/donors.destination.json

```diff
@@ -0,0 +1,578 @@
+      "display_label":"HUTT, Mrs",
+      "ca_entities.idno":"5",
+      "ca_entities.preferred_labels.displayname":"HUTT, Mrs",
+      "ca_entities.preferred_labels.prefix":"Mrs",
+      "ca_entities.preferred_labels.forename":"Mrs",
+      "ca_entities.preferred_labels.middlename":"MRS",
+      "ca_entities.preferred_labels.surname":"Hutt,",
+      "ca_entities.AddressContainer.Address1":"28 Thomas Street",
+      "ca_entities.AddressContainer.TownCity":"NEDLANDS",
+      "ca_entities.AddressContainer.State":"WA",
+      "ca_entities.AddressContainer.PostCode":"6009",
+      "ca_entities.DonorNotes":"Donated in early 1978",
+      "ca_entities.LastEditBy":"IST",
+      "ca_entities.LastEditDate":"19 December 2000"
+    },
+    {
+      "entity_id":"5",
+      "id":"5",
+      "idno":"6",
+      "display_label":"FAIRBAIRN, Miss, and Dr Fairbairn",
+      "ca_entities.idno":"6",
+      "ca_entities.preferred_labels.displayname":"FAIRBAIRN, Miss, and Dr Fairbairn",
+      "ca_entities.preferred_labels.prefix":"Miss",
+      "ca_entities.preferred_labels.forename":"Miss, and Dr Fairbairn",
+      "ca_entities.preferred_labels.middlename":"A",
+      "ca_entities.preferred_labels.surname":"Fairbairn,",
+      "ca_entities.Email":"`",
+      "ca_entities.DonorNotes":"Also donated by Dr Fairbairn in 1937",
+      "ca_entities.LastEditBy":"IST",
+      "ca_entities.LastEditDate":"19 December 2000"
+    },
+    {
+      "entity_id":"6",
+      "id":"6",
+      "idno":"7",
+      "display_label":"CLIFTON, Edmund",
+      "ca_entities.idno":"7",
+      "ca_entities.preferred_labels.displayname":"CLIFTON, Edmund",
+      "ca_entities.preferred_labels.prefix":"Mr",
+      "ca_entities.preferred_labels.forename":"Edmund",
+      "ca_entities.preferred_labels.middlename":"E",
+      "ca_entities.preferred_labels.surname":"Clifton,",
+      "ca_entities.AddressContainer.TownCity":"SOUTH PERTH",
+      "ca_entities.LastEditBy":"DodG",
+      "ca_entities.LastEditDate":"12 July 2002"
+    },
+    {
+      "entity_id":"7",
+      "id":"7",
+      "idno":"8",
+      "display_label":"ROLLAND",
+      "ca_entities.idno":"8",
+      "ca_entities.preferred_labels.displayname":"ROLLAND",
+      "ca_entities.preferred_labels.prefix":"Miss",
+      "ca_entities.preferred_labels.middlename":"N",
+      "ca_entities.preferred_labels.surname":"Rolland",
+      "ca_entities.AddressContainer.Address1":"11 Kanimbla Road",
+      "ca_entities.AddressContainer.TownCity":"NEDLANDS",
+      "ca_entities.AddressContainer.State":"WA",
+      "ca_entities.AddressContainer.PostCode":"6009",
+      "ca_entities.DonorNotes":"Donor's father was assistant Engineer in Chief of WA.",
+      "ca_entities.LastEditBy":"IST",
+      "ca_entities.LastEditDate":"19 December 2000"
+    },
+    {
+      "entity_id":"8",
+      "id":"8",
+      "idno":"9",
+      "display_label":"ANDERSON Nell",
+      "ca_entities.idno":"9",
+      "ca_entities.preferred_labels.displayname":"ANDERSON Nell",
```

Comment:
This should be fixed in the source data. Alternatively we could consistently format the display name as you suggest, but I think it's better to leave it as they have it in the source.